lib/ui/semantics.dart

```diff
@@ -118,6 +120,32 @@ class SemanticsAction {
   /// Paste the current content of the clipboard.
   static const SemanticsAction paste = const SemanticsAction._(_kPasteIndex);
 
+  /// Accessibility focus is moved to the node.
+  ///
+  /// This handler is invoked when the accessibility focus is moved
+  /// to the node annotated with this handler. The accessibility focus is the
+  /// green (on Android with  TalkBack) or black (on iOS with VoiceOver)
+  /// rectangular show on screen to indicate with what element an accessibility
+  /// user is currently interacting with.
+  ///
+  /// The accessibility focus is different from the input focus. The input focus
+  /// is usually held by the element that currently response to keyboard inputs.
+  /// Accessibility focus and input focus can be held by two different nodes!
+  static const SemanticsAction accessibilityFocus = const SemanticsAction._(_kAccessibilityFocus);
```

Comment:
I'd call these "didGainAccessibilityFocus" and "didLoseAccessibilityFocus". Otherwise, the names to not convey temporal information about these actions (I'm assuming these are fired _after_ the physical user action).